tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -141,6 +141,10 @@ def create_parser(product_choices=None):
                                       action="append",
                                       choices=["not-implementing", "backlog", "implementing"],
                                       help="Skip tests that have the given implementation status")
+    # TODO: Remove this when QUIC is enabled by default.
+    test_selection_group.add_argument("--include-quic", action="store_true", default=False,
```

Comment:
Nit; Was `--include-quic` decided on somewhere else? Otherwise I would prefer `--enable-quic` I think

Reply:
No, I changed to `--enable-quic`, which I agree sounds better.